includes/payment-methods/class-upe-payment-gateway.php

```diff
@@ -134,8 +134,9 @@ public function update_payment_intent_ajax() {
 			$payment_intent_id         = isset( $_POST['wc_payment_intent_id'] ) ? wc_clean( wp_unslash( $_POST['wc_payment_intent_id'] ) ) : '';
 			$save_payment_method       = isset( $_POST['save_payment_method'] ) ? 'yes' === wc_clean( wp_unslash( $_POST['save_payment_method'] ) ) : false;
 			$selected_upe_payment_type = ! empty( $_POST['wcpay_selected_upe_payment_type'] ) ? wc_clean( wp_unslash( $_POST['wcpay_selected_upe_payment_type'] ) ) : '';
+			$payment_country           = ! empty( $_POST['wcpay_payment_country'] ) ? wc_clean( wp_unslash( $_POST['wcpay_payment_country'] ) ) : '';
```

Comment:
Since we're defaulting to null in the function parameters, I was wondering if we should use null here as well? Or the other way around and use the empty string as the default parameter value? I don't think it will make any difference, but was just thinking it could be nice to keep with the same default in each scenario?

Reply:
Thanks for catching that @bborman22! I missed it after switching between null and empty string a couple of times.
*Addressed in b205d96*